apps/benchmark/src/implementations/merge-styles-v7/View.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { mergeStyleSets } from '@uifabric/merge-styles';
+
+const getClassNames = () => {
+  return mergeStyleSets({
+    root: {
```

Comment:
```suggestion
  return mergeStyles({
```

This should be just `mergeStyles` call, correct?